tests/api/requestargs_test.py

```diff
@@ -20,15 +20,17 @@ class RequestArgsTestCase(TestCase):
     @setup
     def setup_args(self):
         self.args = {
-            'number':   ['123'],
-            'string':   ['astring'],
-            'boolean':  ['1'],
-            'datetime': ['2012-03-14 15:09:26'],
+            b'number':   [b'123'],
+            b'string':   [b'astring'],
+            b'boolean':  [b'1'],
+            b'datetime': [b'2012-03-14 15:09:26'],
```

Comment:
Making part of them be bytes and part of them be str might be more comprehensive for testing?

Reply:
yeah, but this is emulating twisted's args map and that only ever contains keys/values as bytes